content/integrations/jenkins.md

```diff
@@ -9,29 +9,31 @@ newhlevel: true
 ## Overview
 
 {{< img src="integrations/jenkins/integration-jenkins-overview.png" alt="Jenkins Plugin" >}}
-A Jenkins plugin used to forward metrics, events, and service checks to an account at Datadog, automatically.
+A Jenkins plugin used to forward metrics, events, and service checks to an account at Datadog, automatically. 
```

Comment:
I think we can tighten up the whole Overview. In particular the **Note** is too wordy.

---
This [Jenkins plugin](https://github.com/DataDog/jenkins-datadog-plugin) sends your build and deployment events to Datadog. From there, you can overlay them onto graphs of your other metrics so you can identify which deployments really affect your application's performance and reliability—for better or worse.

**Note: The Jenkins check for the Datadog Agent is deprecated.** Use the Jenkins plugin.